checksum/host/Makefile

```diff
@@ -17,7 +17,7 @@ OBJEXT      := o
 #Flags, Libraries and Includes
 PROFILE     := $(if $(DPU_TARGET_PROFILE),-DDPU_PROFILE=$(DPU_TARGET_PROFILE))
 CFLAGS      := -std=c11 -Wall -O3 -g -DDPU_TYPE=$(DPU_TARGET_TYPE) $(PROFILE)
-LIB         := -ldpu -ldpucni -lpthread -L$(UPMEM_HOME)/usr/lib
+LIB         := -ldpu -lpthread -L$(UPMEM_HOME)/usr/lib
```

Comment:
Do we need the explicit link to the `pthread` library ?

Reply:
not sure that the indirect linking through libdpu is better,
but removed